apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.js

```diff
@@ -142,8 +165,9 @@ window.PLOrderBlocks = function (uuid, options) {
       // when the user stops interacting with the list
       let leftDiff = calculateIndent(ui, ui.item.parent());
       ui.item[0].style.marginLeft = leftDiff + 'px';
-      setAnswer();
+      $(dropzoneElementId)[0].style.background = null;
 
+      setAnswer();
       correctPairing(ui);
```

Comment:
after moving the creation of the lines to the `create` function, this change should be reverted.

Reply:
Code changed as suggested. You're right, having the lines be persistent makes more sense.